src/update_objects.c

```diff
@@ -7856,7 +7859,7 @@ void func_80086700(s32 objectIndex) {
     set_obj_origin_offset(objectIndex, 0.0f, 0.0f, 0.0f);
     set_obj_direction_angle(objectIndex, 0U, 0U, 0U);
     gObjectList[objectIndex].unk_084[1] = 0x0200;
-    func_80072488(objectIndex);
+    reset_timer_go_next_state(objectIndex);
```

Comment:
`reset_timer_run_next_state`?

Reply:
or set_next_state

Reply:
initiate_next_state

Reply:
Run_next_state are not correct because it don't call a function

Reply:
Initialise_next_state would be better

Reply:
initialize

Reply:
set_next_animation_state ?

Reply:
advance_object_state?

Reply:
The timer it's not for animation it's for object to know when he go to the next state